invenio_files_rest/serializer.py

```diff
@@ -50,3 +51,45 @@ def json_serializer(data=None, code=200, headers=None):
     # ETag needed?
     # response.set_etag(str(record.model.version_id))
     return response
+
+
+def empty_if_none(f):
+    """Wrap a serialize to handle when data is None."""
+    @wraps(f)
+    def decorator(data=None, code=200, headers=None):
+        data = data if not data else f(data, code, headers)
+        return json_serializer({'json': data}, code, headers)
+    return decorator
+
+
+@empty_if_none
+def bucket_collection_serializer(data=None, code=200, headers=None):
+    """Serialize BucketCollectionResource responses."""
+    def serialize(bucket):
```

Comment:
We should try to match as close as possible the S3 REST API: http://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html